WooCommerce/Classes/ViewRelated/Dashboard/Settings/CardReadersV2/Tap to Pay Education/TapToPayEducationViewModel.swift

```diff
@@ -27,7 +27,7 @@ final class TapToPayEducationViewModel: ObservableObject {
     let cardPresentPaymentsOnboardingUseCase: CardPresentPaymentsOnboardingUseCaseProtocol
     let siteID: Int64
 
-    private let onDismiss: () -> Void
+    var onDismiss: () -> Void
```

Comment:
`onDismiss` implies a notification that it is being dismissed, but I think this is actually a `dismissAction` –  calling it dismisses the view, and then the other `onDismiss` (which was passed in to the hosting controller) is called to notify something that it was dismissed as well.